homework_03/07_gergana_roeva/hw03.py

```diff
@@ -29,25 +26,21 @@ def distance(self, other_circle):
         return abs(sqrt((other_circle.center.x - self.center.x)**2 + (other_circle.center.y - self.center.y)**2))
 
     def are_same(self, other_circle):
-        return((other_circle.r == self.r) and self.same_center(other_circle))
-
-    def are_not_touching(self, other_circle):
-        if(self.r + other_circle.r < self.distance(other_circle)):
+        if(abs(self.r - other_circle.r) == self.distance(other_circle)):
             return True
-
+        
         if(abs(self.r - other_circle.r) > self.distance(other_circle)):
             return True
+
+        return((other_circle.r == self.r) and self.same_center(other_circle))
         
-        return False
+
+    def are_not_touching(self, other_circle):
+        return(self.r + other_circle.r < self.distance(other_circle))
     
     def are_touching(self, other_circle):
-        if(self.r + other_circle.r == self.distance(other_circle)):
-            return True
+        return(self.r + other_circle.r == self.distance(other_circle))
 
-        if(abs(self.r - other_circle.r) == self.distance(other_circle)):
-            return True
-        
-        return False
 
     def find_relative_position(self, other_circle):
```

Comment:
I like that you made everything into separate functions. Makes the code very easy to read here.